google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/BatchReadOnlyTransaction.java

```diff
@@ -71,7 +71,7 @@
    *       long singerId = results.getLong(0);
    *       String firstName = results.getString(1);
    *       String lastName = results.getString(2);
-   *       System.out.println("P2 [" + singerId + "] " + firstName + " " + lastName);
+   *       System.out.println("P[" + singerId + "] " + firstName + " " + lastName);
```

Comment:
curious what does P denote here

Reply:
I copied this from some code sample. I'll take it out.